content/docs/policy/README.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Policy
+description: 'Rules for guiding certificate self-service.'
```

Comment:
This description doesn't make sense to me. 
I am a new small-scale user (not-multi-tenant) who wants to do the best thing by ensure that all the Certificates or csi-driver settings are using ECDSA keys and that private key rotation is set to always.
Someone sends me a link ending `.../policy/`  and it unfurls in Slack with the description "Rules for guiding certificate self-service".
I might assume that I'd been sent the wrong link.